.travis.yml

```diff
@@ -19,6 +19,7 @@ before_script:
 script:
   - yarn lint
   - yarn build
+  - yarn test
```

Comment:
I would suggest to do yarn test before yarn build. It should go from the faster to the slower and I think that the building step is the slowest task from the 3.